about.html

```diff
@@ -0,0 +1,94 @@
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Join</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Sponsor</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
+          <small class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur recusandae voluptatibus corporis ipsa temporibus minus incidunt deleniti in earum enim.</small>
+          <small class="contact-us">Please contact us via Email for any further questions about TMC production conference</small>
+          <a class="contact-link" href="#">Tritonemusic.co@enquiries.org</a>
+        </section>
+        <section class="two">
+            <h3>Tritone Music Co Producers Conference Logo</h2>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit labore saepe dolores dolorem laboriosam pariatur eveniet aliquid? Ipsum, architecto possimus.</p>
+            <figure class="logo-img"></figure>
+        </section>
+        <section class="three">
+            <h3>See the past TMC production Conference</h3>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore quisquam enim, atque nesciunt doloremque praesentium!</p>
+            <div class="gallery">
+                <div class="image one">
+                    <h3 class="year">2022</h3>
+                    <span class="location">TMC production conference in Accra</span>
+                </div>
+                <div class="image two">
+                    <h3 class="year">2021</h3>
+                    <span class="location">TMC production conference in Kumasi</span>
+                </div>    
+            </div>
+        </section>
+        <footer>
+            <div class="partners-info">
+              <div class="partner-heading">
+                <h2>Partner</h2>
+                <div class="divider"></div>
+              </div>
+              <ul class="partners">
+                <li><a href=""><img src="assets/images/airbnb.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/googlekorea.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/mozilla.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/naver.png" alt="" class="partner-icon"></a></li>
+              </ul>
```

Comment:
- [ ] In your desktop version in about page  the partners section is not exist but there is just the last footer with different background color so please remove the partner section and change the background color to the last footer to match the design 🔥💯🙏🏻♻

- Your design
![image](https://user-images.githubusercontent.com/58553711/212223926-cce63f91-c471-464f-969f-cb494f622fdd.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/212223964-1eeb459f-621e-4176-8853-b90d48f458c3.png)